README.md

```diff
@@ -13,6 +13,11 @@ Admins in their turn can:
  - delete users from the store's database;
  - view all orders placed at the store; 
  - edit the store's stock of instruments.
+ 
+## Project Structure
+The project has an N-tier structure and consists of the database layer, the DAO layer for interaction with the database, the service layer which contains the business logic, and the presentation layer.
+Servlets are used to receive and respond to client requests; filters control access to the store's functionality; and the presentation layer is implemented with JSP and Bootstrap. The DAO layer includes two implementations - one for the JDBC, which is used for work with a database; and one where data is stored locally in Lists. 
+This project also includes custom-made annotations and an injector, which utilizes Reflection API. 
 
```

Comment:
Was not able to write next, but I think MySQL Connector is now enough, better write MySQL RDMS and JDBC